composer.json

```diff
@@ -17,12 +17,12 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "ably/ably-php": "^1.1",
-        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0 || ^11.0",
+        "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0 || ^11.0 || ^12.0",
```

Comment:
Laravel's 6, 7 and 8 have been EOL for > 2 years now, is this a good time to retire support?

Reply:
We are just running tests against it, if there are customers who are using old version of laravel with pusher or socket.io and if they want to switch to ably, they should be able to do it.

Reply:
Same goes for laravel-echo. I remember we started with weekly downloads of 500, but now it's 3000+ => https://www.npmjs.com/package/@ably/laravel-echo
I think MRR of laravel-echo is almost same as ably-flutter. I see lot of future potential for laravel-echo

Reply:
Sure, as long as it's not a blocker to us maintaining the package effectively